src/actions/stringTypes.ts

```diff
@@ -1,4 +1,4 @@
-const types = {
+const stringTypes = {
```

Comment:
please change the name here, we called this `actionTypes` and also, this needs to be a `ts` right?